platform/iphone/app_delegate.mm

```diff
@@ -507,90 +110,9 @@ - (void)drawView:(GLView *)view;
 
 		default: {
 			if (OSIPhone::get_singleton()) {
-				// OSIPhone::get_singleton()->update_accelerometer(accel[0], accel[1],
```

Comment:
We also have a separate motion manager class to be solely responsible for updating these values